lib/server/model.js

```diff
@@ -167,11 +181,30 @@ class RangerModel {
     return attrs;
   }
 
+  _fbAttrs(query) {
+    assert(query);
+    assert(RangerModel.hasReference(query),
+      'database does not hold location of reference .fa file');
+    // '-c' option allows freebayes to read from stdin ie. from pipe
+    let attrs = ['-c', '-f', query.reference];
+    if (query.region) {
+      attrs.push('-r', query.region);
+    }
+    LOGGER.debug('freebayes attrs: ' + attrs);
+    return attrs;
+  }
+
   _getFile(query, destination, endResponseClback) {
     const view = child.spawn(SAMTOOLS_COMMAND, this._stViewAttrs(query));
     view.title = 'samtools view';
+    let processes = [view];
+    if (query.format === 'VCF') {
+      const freebayes = child.spawn(FREEBAYES_COMMAND, this._fbAttrs(query));
```

Comment:
Maybe something like
`_scheduleVCF = (processes) => { const freebayes ...; process.push(freebayes); }`
then use in both places to avoid code repetition